crates/wasm/fs/public/directory.rs

```diff
@@ -152,7 +157,7 @@ impl PublicDirectory {
         time: &js_sys::Date,
```

Comment:
Unfortunately I can't comment on line 148 because it wasn't changed in this PR. But why do we have to pass a `&mut self`. We're not really mutating the `PublicDirectory`, so I'm not sure why it would be needed :thinking:

Reply:
You are right. They are residues of refactoring. Removed.